app/views/landing_page/landing_page.erb

```diff
@@ -29,7 +29,7 @@
   <meta charset="utf-8">
   <meta name="viewport" content="width=device-width, initial-scale=1">
 
-  <title>Custom Landing Page</title>
+  <title><%= settings["marketplace_name"]["value"] %></title>
```

Comment:
Define a new top level property that would hold global data and link to marketplace data from there. Other stuff that would come under the top level property would be things like meta tags for SEO, i.e. things that cannot be disabled and don't have a position like sections.